### DISCUSSION TITLE
Review added at repo-reviews.github.io

### DISCUSSION THREAD

**Discussion by repo-reviews** [openai/whisper#1533] [Q&A]
### Thank you for building **whisper**!

@richardanderson001 created a review titled:

*Well built with easy install*

on [repo-reviews.github.io](https://repo-reviews.github.io) to share their experience using **whisper**.

[link to review](https://repo-reviews.github.io//reviews/2023-07-18_richardanderson001_openai_whisper)

If you would like to help your super-users share their experiences using your repo, add a [badge](https://github.com/repo-reviews/repo-reviews.github.io#add-badges) to your README.md.

We hope that sharing these experiences helps your users **increase their productivity**.

--
Please be kind,
I’m a human!

**Comment by phineas-pta** [comment]
instead of adding review feature, they made a bot to spam issues to repo 👍
